admin/src/js/services/select2-search.js

```diff
@@ -147,12 +206,43 @@ angular.module('inboxServices').factory('Select2Search',
           }
         }
 
-        return resolution.then(function() {
-          selectEl.trigger('change');
+        return resolution.then(function() { //NoSONAR
+          $timeout(() => { //NoSONAR
+            selectEl.trigger('change');
+          }, 1000);
```

Comment:
I think we should figure out why this timeout is needed. I'm not comfortable with keeping this at all.

Reply:
This seems what's mainly left. I will dedicate tomorrow morning to it.